README.md

```diff
@@ -36,7 +36,7 @@ application dependency:
 
 ```elixir
 def application do
-  [applications: [:httpoison]]
+  [extra_applications: [:httpoison]]
```

Comment:
As for the comment about 1.4 and not needing to include the app. It appears to not hurt to list the packages and as such I would also just leave it as it is more explicit and makes it easier to know what is actually being used.